### DISCUSSION TITLE
Addon for reading localization translations

### DISCUSSION THREAD

**Discussion by Apoerin** [storybookjs/storybook#16923] [Help]
Is there any working addon for internalization or I can do it without addon? 
Every addon I tried doesn't return values of translations, listed in `json`, just `button.save` for example.
So far I tried 
```
    "storybook-addon-intl": "^2.4.1"
    "storybook-react-intl": "^0.9.2"
```
Any suggestions would be appreciated

**Comment by shilman** [comment]
cc @stevensacks

**Reply by Apoerin** [reply]
@shilman I already made an issue there, in @stevensacks repo 6 days ago, but got no answer at all

**Comment by Apoerin** [comment]
so, no answer at all? No one ever had the same problem?
